unity-renderer/Assets/DCLPlugins/EquippedEmotes/EquippedEmotesPlugin.cs

```diff
@@ -0,0 +1,106 @@
+using DCL.Helpers;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace DCL.EquippedEmotes
+{
+    /// <summary>
+    /// Plugin feature that initialize the Equipped Emotes feature.
+    /// </summary>
+    public class EquippedEmotesPlugin : IPlugin
```

Comment:
I would call this `EquippedEmotesInitializer` or  `EquippedEmotesStorageHandler`

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2078/commits/01c8aff3a2548f8f13f86d909574acb894cad1c8.